### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by pwright**
## Motivation
testing custom walkthroughs

## Why
seems to have an incorrect reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
-To run an example customisation you need an exiting OpenShift cluster with integreatly installed.
+To run an example customisation you need an existing OpenShift cluster with integreatly installed.
 
 Next you need to get the available inventory file from the manifest secret
 
 ```
 (mac osx)
 
-oc get secret inventory -n webapp --template '{{index .data "generated_inventory"}}'  | base64 -D > inventory
+oc get secret manifest -n webapp --template '{{index .data "generated_inventory"}}'  | base64 -D > inventory
```
Comment (pwright) [inline README.md:13]: oc get secret inventory -n webapp --template '{{index .data "generated_inventory"}}'  | base64 -d > inventory

gives the following error

Error from server (NotFound): secrets "inventory" not found


Guessing solution above

Comment (david-martin) [reply README.md:13]: @maleck13 is the secret going to be 'manifest' or 'inventory'

Comment (maleck13) [reply README.md:13]: @finp Is this from master. There are two secrets. The manifest and the inventory

**Comment by pwright** [comment]
@maleck13  thought I understood what was going on, closing, follow up with https://issues.jboss.org/browse/INTLY-333

**Pull request closed without merging**